tests/mfx_dispatch/linux/mfx_dispatch_test_mocks.h

```diff
@@ -105,7 +105,7 @@ class MockCallObj : public CallObj
     char* FeedEmulatedPluginsCfgLines(char * str, int num, FILE * stream);
     mfxStatus ReturnMockPlugin(mfxPluginUID uid, mfxPlugin* pPlugin)
     {
-        pPlugin = &m_mock_plugin;
+        *pPlugin = m_mock_plugin;
```

Comment:
pointer check is required, pPlugin might be nullptr here. Other solution - change function signature to accept reference

Reply:
This function signature is determined by CreatePlugin function signature. I will add nullptr check for pPlugin. Changing of pPlugin pointer and further check are impossible.